contracts/voting-verifier/src/query.rs

```diff
@@ -1,58 +1,37 @@
-use axelar_wasm_std::operators::Operators;
-use axelar_wasm_std::voting::{PollStatus, Vote};
+use axelar_wasm_std::{
+    operators::Operators,
+    voting::{PollStatus, Vote},
+    VerificationStatus,
+};
 use connection_router::state::{CrossChainId, Message};
-use cosmwasm_schema::cw_serde;
 use cosmwasm_std::Deps;
 
 use crate::error::ContractError;
 use crate::state::{self, Poll, PollContent, POLLS, POLL_MESSAGES, POLL_WORKER_SETS};
 
-#[cw_serde]
-pub enum VerificationStatus {
-    Verified,
-    FailedToVerify,
-    InProgress,  // still in an open poll
-    NotVerified, // not in a poll
-}
-
-pub fn is_verified(
+pub fn messages_status(
```

Comment:
It's `MessageStatus` at other places though https://github.com/axelarnetwork/axelar-amplifier/blob/24f4a301e5636f0c51497d17a5e8ffc7164a41e3/contracts/voting-verifier/src/msg.rs#L65. Need to be consistent.